Resources/Prototypes/Ronstation/Research/civilianservices.yml

```
@@ -0,0 +1,13 @@
+- type: technology
+  id: Cloning
+  name: research-technology-cloning
+  icon:
+    sprite: /Structures/Machines/scanner.rsi
```

Please delete the “/“ in front of the “Structures”; it’s causing errors.

```suggestion
    sprite: Structures/Machines/scanner.rsi
```